pinot-query-runtime/src/test/resources/queries/Aggregates.json

```diff
@@ -0,0 +1,297 @@
+            1.5,
+            "c",
+            false
+          ],
+          [
+            175,
+            1.75,
+            "c",
+            true
+          ]
+        ]
+      }
+    },
+    "queries": [
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "comments": "result error:average doesn't work because we round up but h2 round down for integer",
+        "description": "average int",
+        "sql": "SELECT avg(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "average double",
+        "sql": "SELECT avg(double_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "average double with filter",
+        "sql": "SELECT avg(double_col) FROM {tbl} WHERE double_col >= 100"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "sum double",
+        "sql": "SELECT sum(double_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "sum int",
+        "sql": "SELECT sum(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "max double",
+        "sql": "SELECT max(double_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "max int",
+        "sql": "SELECT max(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "count int",
+        "sql": "SELECT count(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "count double",
+        "sql": "SELECT count(double_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "count, sum group by order by",
+        "sql": "select string_col, count(int_col), sum(double_col) from {tbl} group by string_col order by string_col;"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "min, max",
+        "sql": "SELECT min(int_col), max(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "comments": "plan error:Unsupported SQL aggregation kind: {}. Only splittable aggregation functions are supported! [SINGLE_VALUE]",
+        "ignored": true,
+        "description": "outer aggregate",
+        "sql": "select  (select max((select i.int_col from {tbl} i where i.int_col = o.int_col))) from {tbl} o;"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "description": "nested",
+        "comments": "this should throw an error",
+        "sql": "SELECT min(max(int_col)) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "description": "nested",
+        "comments": "this should throw an error",
+        "sql": "SELECT (SELECT max(min({int_col})) FROM {tbl}) from {tbl};"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "comments": "parsing error:ORDER BY unexpected",
+        "description": "filter",
+        "sql": "SELECT min(double_col ORDER BY int_col) FROM {tbl}"
```

Comment:
are these failed tests expected failures or tests that should fail but don't? if they should fail but don't we should add expectedException instead of ignoring them

Reply:
Added the expectedException for the one we expect it to fail. For others, we don't expect them to fail. I copied those test cases from psql test.